ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -266,14 +266,19 @@ export const ConceptHomepage = withCurrentWorkspace()(
       }
     }
 
-    triggerSearch(e) {
+    handleChange(e) {
+      this.setState({currentSearchString: e});
+    }
+
+    handleSubmit(e) {
```

Comment:
Naming nit: since this method is actually called on every keypress, maybe a better name would be "maybeHandleSubmit"? Combined with my nitpick above, we end up with an awkwardly long, but very precise, name of "maybeHandleSearchSubmit". Your call whether that's any more readable than what's here.